### PR TITLE
Add tests to the ShellItemWrapper class.

### PR DESCRIPTION
**Pull request by juandausa**
Add tests.

### CODE REVIEW

File: packages/file_selector/file_selector_windows/test/shell_item_wrapper_test.dart [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import 'package:file_selector_windows/src/shell_item_wrapper.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:win32/win32.dart';
+import 'shell_item_array_mock.dart';
+
+void main() {
+  final ShellItemWrapper shellItemWrapper = ShellItemWrapper();
+
+  setUp(() {});
```
Comment (eugerossetto) [inline packages/file_selector/file_selector_windows/test/shell_item_wrapper_test.dart:17]: Is this needed?

File: packages/file_selector/file_selector_windows/test/shell_item_wrapper_test.dart [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  tearDown(() {});
```
[the remark this answers is not in the document]
Comment (juandausa) [reply packages/file_selector/file_selector_windows/test/shell_item_wrapper_test.dart]: Removed, thanks.

File: packages/file_selector/file_selector_windows/test/shell_item_wrapper_test.dart (hunk 1 above)
Comment (juandausa) [reply packages/file_selector/file_selector_windows/test/shell_item_wrapper_test.dart]: Removed.

File: packages/file_selector/file_selector_windows/test/shell_item_wrapper_test.dart [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        isA<IShellItemArray>());
+    free(ptrComObject);
+  });
+
+  test('getCount invokes shellItemArray getCount', () {
+    final FakeIShellItemArray shellItemArray = FakeIShellItemArray();
+    final Pointer<Uint32> ptrNumberOfItems = calloc<Uint32>();
+
+    shellItemWrapper.getCount(ptrNumberOfItems, shellItemArray);
+    expect(1, shellItemArray.getCountCalledTimes());
```
Comment (juandausa) [inline packages/file_selector/file_selector_windows/test/shell_item_wrapper_test.dart:39]: This expect should be the other way around.

Comment (juandausa) [reply packages/file_selector/file_selector_windows/test/shell_item_wrapper_test.dart]: Fixed.

**Pull request merged**